### DISCUSSION TITLE
Laravel Volt Support

### DISCUSSION THREAD

**Discussion by 4m1n0s** [filamentphp/filament#7307] [Ideas]
What do you think about using Filament Forms/Tables .... inside volt component?

<img width="576" alt="image" src="https://github.com/filamentphp/filament/assets/2763884/22631bb5-2c0b-4109-81a6-c68d049f90bb">

**Comment by danharrin** [comment]
Yup working on it soon!

**Reply by binaryfire** [reply]
Awesome. Will it work the other way around too? Eg. including a Volt component in an infolist with View::make?

**Reply by fikurimax** [reply]
any updates?

**Comment by alexworldbr** [comment]
Anybody used VOLT with FILAMENTPHP?

**Reply by alexworldbr** [reply]
Do we have same sample how to do it?

**Comment by agelospanagiotakis** [comment]
any news on this one ? is it possible ?
